src/tests/dictionaryConcepts/components/ConceptNameRows.test.jsx

```diff
@@ -5,7 +5,7 @@ import ConceptNameRows from '../../../components/dictionaryConcepts/components/C
 const props = {
   pathName: {
     language: 'en',
-  }
```

Comment:
Is this a formatting change?

Reply:
Yes it is.

Reply:
All formatting changes that are not in line with your PR task should be done in a different PR.

Reply:
That is true!